### PR TITLE
Editorial: Add Contacts API history note

### PR DESCRIPTION
**Pull request by anssiko**
Fix https://tantek.com/2020/026/t2

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/contact-picker/pull/58.html" title="Last updated on Feb 6, 2023, 12:45 AM UTC (2fb82c7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/contact-picker/58/996229a...2fb82c7.html" title="Last updated on Feb 6, 2023, 12:45 AM UTC (2fb82c7)">Diff</a>

### CODE REVIEW

**Comment by anssiko** [comment]
Related TAG discussion: https://github.com/w3ctag/tag.w3.org/issues/22

Further work on top of this would be to expand this note with a (brief) explanation on what differentiates this proposal from the past efforts that did not gain traction. The spec editors are in the best position to do so.

**Review by marcoscaceres — COMMENTED** [review]
lgtm, though I don't know if we should claim anything about the "rich history"... I think the community just generally got better at designing these things, even if those APIs didn't get any look at when this new API was designed.

@rayankans, is that a fair assessment? 

Thus, I'd personally be inclined to drop "and this API strives to learn from this rich history" and just acknowledge the past... maybe we can should move this into an Acknowledgments section?

**Comment by marcoscaceres** [comment]
Oh, I found this (in the README): https://github.com/w3c/contact-picker/pull/31/

I wonder if we should just migrate that over from the README?

**Comment by anssiko** [comment]
Updated to explain what differentiates this proposal from the past.

**Comment by rayankans** [comment]
> I think the community just generally got better at designing these things, even if those APIs didn't get any look at when this new API was designed.

Kind of, all of the previous attempts were designed with full & perpetual access (gated by a permission) in mind, which is far from ideal, and contact edit access, which is not a use-case we found any legitimate interest in. These fundamental differences made us decide to start a new proposal rather than build on top of the previous ones.

> maybe we can should move this into an Acknowledgments section?

+1, that seems like a more appropriate place

**Comment by anssiko** [comment]
Moved to an Acknowledgments section.

**Comment by anssiko** [comment]
@tantek friendly ping. Do you think this PR will address your issue documented in https://tantek.com/2020/026/t2

File: index.bs
Comment (marcoscaceres) [inline index.bs:463]:
```suggestion
sections of the earlier attempts.

```

**Review by marcoscaceres — APPROVED** [review]

**Comment by anssiko** [comment]
Thanks @marcoscaceres for the improvements and review.

@tantek any last-minute suggestions before we merge this? I think it is helpful to have this text in the spec so future generations will find the prior work and can learn from it. Thanks for raising this issue in your blog to begin with.

**Comment by anssiko** [comment]
I'll merge this Acknowledgments update and we will capture any additional feedback in subsequent PRs.

@tantek if this PR does not address your initial concern, please let us know.

**Pull request merged**
